config.sample.yml

```diff
@@ -58,6 +58,10 @@ influxdb_publisher:
   bucket: ruuvi
   # Measurement name to use
   measurement: ruuvi_measurements
+  # Additional tags to add to a series
+  additional_tags:
+    mytag: myvalue
```

Comment:
I think it's better to comment out these two "example tags" to prevent them from being unnecessarily added to the actual data if a user just copies the entire sample config and changes only what they strictly need to change from a "bare minimum configuration", similar to `mqtt_publisher.homeassistant_discovery_prefix`

Reply:
Yes, that makes sense. I've updated the config example accordingly.